src/mumble/Log.ui

```diff
@@ -127,7 +99,20 @@
         </property>
        </widget>
       </item>
-      <item row="1" column="1">
+      <item row="0" column="0">
+       <widget class="QCheckBox" name="qcbEnableTTS">
```

Comment:
What about simply `qcbTTS`?

Reply:
I actually think `qcbEnableTTS` is more accurate and should therefore be preferred.